assets/src/application/ui/layout/descriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { DescriptionListProps } from './types';
+import './styles.scss';
+
+const DescriptionList: React.FC<DescriptionListProps> = ({ dataSource }) => {
+	return (
+		<dl className='ee-description-list-grid'>
+			{dataSource.map(({ description, term }) => {
+				return (
+					<div>
+						<dt>{term}</dt>
+						<dd>{description}</dd>
+					</div>
```

Comment:
This can be wrapped in `div`, based on MDN:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

![image](https://user-images.githubusercontent.com/1477453/76968118-0cf63f00-6931-11ea-9be1-46de63ae3411.png)